src/App/App.tsx

```diff
@@ -1,187 +1,105 @@
-import React, { Component, ReactElement } from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'react-emotion';
+import Box from '@material-ui/core/Box';
 import isNil from 'lodash/isNil';
-import 'normalize.css';
```

Comment:
`import 'normalize.css';` we need this

Reply:
I have created the ResetCss file and I am also using the Baseline file from MUI 🙃 so I believe it is not necessary to have a normalize css dependency